### PR TITLE
WDB-129/Folder creation improvements

### PR DESCRIPTION
**Pull request by tmaciulis22**
Move folder creation to upload chunks endpoint, some other minor code changes

### CODE REVIEW

**Review by salmonela7 — APPROVED** [review]
LGTM

File: backend/BusinessLogic/Services/VideoService/VideoService.cs [hunk 1]
```diff
@@ -110,8 +112,15 @@ public async Task DeleteVideo(Video video, Guid userId)
         public void CreateUserVideoDirectory(Guid userId)
         {
             string userPath = Path.Combine(_uploadPath, userId.ToString());
-            Directory.CreateDirectory(userPath);
-            Directory.CreateDirectory(Path.Combine(userPath, "/Snapshots"));
+            if (!Directory.Exists(userPath))
+            {
+                Directory.CreateDirectory(userPath);
+            }
+            string snapshotsPath = Path.Combine(userPath, "/Snapshots");
```
Comment (AArlauskas) [inline backend/BusinessLogic/Services/VideoService/VideoService.cs]: forward slash is not needed

File: backend/BusinessLogic/Services/VideoService/VideoService.cs [hunk 2]
```diff
@@ -182,7 +191,7 @@ await using (var stream = new FileStream(zipCreatePath, FileMode.Open))
             return memory;
         }
 
-        public async Task MarkVideoForDeleteion(Video video)
+        public async Task MarkVideoForDeletion(Video video)
```
Comment (AArlauskas) [inline backend/BusinessLogic/Services/VideoService/VideoService.cs:194]: Thanks

**Review by AArlauskas — APPROVED** [review]
One thing to remove, but LGTM

**Pull request merged**